### PR TITLE
Add a build script for creating the minified JSON

### PR DESCRIPTION
**Pull request by jsantell**

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var fs = require('fs');
```
Comment (cvan) [inline scripts/build.js]: can use `const`

File: scripts/build.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+var path = require('path');
+
+var inputFile = path.join(__dirname, '..', 'dpdb-formatted.json');
+var outputFile = path.join(__dirname, '..', 'dpdb.json');
+var formattedJSON = fs.readFileSync(inputFile, 'utf8');
+
+// Use JSON.stringify/parse to ensure that the JSON is valid
+// rather than just stripping out space/newline chars
+var minifiedJSON = JSON.stringify(JSON.parse(formattedJSON));
```
Comment (cvan) [inline scripts/build.js]: do you want to format this or no (like with 2 spaces)?

**Review by cvan — CHANGES_REQUESTED** [review]

File: scripts/build.js (hunk 2 above)
Comment (jsantell) [reply scripts/build.js]: The original is formatted already, so makes sense for the other version to be minified, no? Maybe the names should just be dpdb.json and dpdb.min.json or something?

Comment (cvan) [reply scripts/build.js]: minified by default is fine; we can add `.min` and/or `.max` later if we need

**Comment by jsantell** [comment]
@cvan updated s/var/const!

**Pull request merged**